### PR TITLE
US995010: EVE-16227 Bump up to major version

### PR DESCRIPTION
**Pull request by raghavendran-gopalakrishnan**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=995010

### CODE REVIEW

**Comment by raghavendran-gopalakrishnan** [comment]
Along with this, I cleaned up remaining HPE references in the code. But we still have @hpe email ids https://github.com/CAFAudit/audit-service/blob/ae8066dabeb7a2c5c451e43fa25fd5ef4553ec34/docs/_config.yml#L2 lying around in markdowns.

**Review by dermot-hardy — APPROVED** [review]

File: docs/pages/en-us/contract/swagger.yaml [hunk 1]
```diff
@@ -1,132 +1,123 @@
...
+
 swagger: "2.0"
-################################################################################
-#                              API Information                                 #
-################################################################################
-# Provides metadata about the API.
 info:
-  version: v1
-  title: Audit Management Service
+  title: "Audit Web Service"
```
Comment (dermot-hardy) [inline docs/pages/en-us/contract/swagger.yaml:19]: What is going on with this contract?  Why is it changing so radically?

Comment (raghavgopal) [reply docs/pages/en-us/contract/swagger.yaml:19]: The swagger.yaml present in under docs was obsolete. In spite of the swagger not being referenced anywhere in the Docs page, I replaced that with the latest one.

Comment (dermot-hardy) [reply docs/pages/en-us/contract/swagger.yaml:19]: Ok

**Pull request merged**